docker-compose.yml

```diff
@@ -9,7 +9,14 @@ web:
   volumes:
     - .:/thredded
   working_dir: /thredded
-  command: ./bin/docker_command.sh
+  environment:
+    DB: postgresql
+    DB_USERNAME: root
+    DB_PASSWORD:
+  command: >
+    bash -c 'script/wait-for-tcp $THREDDED_DB_1_PORT_5432_TCP_ADDR $THREDDED_DB_1_PORT_5432_TCP_PORT &&
+             script/migrate &&
+             rake dev:server'
```

Comment:
Fancy. Wait until the host and port are accepting connections before running migrate and serve? Damn fancy. Love it.